mxcube3/routes/Login.py

```diff
@@ -42,21 +44,36 @@ def login():
 
     try:
         login_res = limsutils.lims_login(loginID, password)
-        inhouse = limsutils.lims_is_inhouse(login_res)
 
+        if login_res['status']["code"] is not "ok":
+            return deny_access("Could not authenticate")
+
+        inhouse = limsutils.lims_is_inhouse(login_res)
         info = {"valid": limsutils.lims_valid_login(login_res),
                 "local": is_local_host(),
                 "existing_session": limsutils.lims_existing_session(login_res),
                 "inhouse": inhouse}
 
-        _users = logged_in_users(exclude_inhouse=True)
+        _users = logged_in_users(exclude_inhouse=False)
+
+        common_proposal = False
```

Comment:
Whats this `common_proposal` ?

Reply:
two different users can login only if they are in the same proposal only, defined by this `common_proposal`